README.md

```diff
@@ -23,6 +26,20 @@ See our [Contribution Guidelines](CONTRIBUTING.md)
 
 Building an MVP with base functionality
 
+* Sign up flow (for new users withoug account)
```

Comment:
this looks like, something got messed up here.

Reply:
Looks like a merged PR fixed that typo. See: https://github.com/AndroidDev-social/MastodonCompose/pull/29/commits/f1c57c8ff2fde48b2e47d9139188706c004d46cc

Reply:
I think I accepted the wrong changes